docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -30,6 +31,40 @@ All modules must have the following sections defined in this order:
 
 
 
+Copyright
+----------------------
+
+The first 3 lines of every module should look the same. The first line is the
+shebang, but the next 2 lines are the important bit: copyright statement and
+license.
```

Comment:
Don't mix code with copyright and license.  Also don't use absolute line numbers.  A coding declaration also gets thrown in here in some cases and there can be more than one copyright line.  I think we should have copyright and license in separate subsections.